### DISCUSSION TITLE
parse_latex converting `-1` to `-1*1`

### DISCUSSION THREAD

**Discussion by gerryfletch** [sympy/sympy#24031] [Q&A]
Hello!

I'm trying to decide whether a formula is in its simplest form, and a hurdle I'm running into is that formulas of the form `x - n` expands to `-1 * n` for any negative constant. Examples:

```python
parse_latex(-1) # -1       <- works as intended
parse_latex(3-1) # 3 - 1*1
parse_latex(3x-1) # 3*x - 1*1
parse_latex("-3 + x") # x - 3
```

I'm not sure why it's adding in `-1*`. I can't replicate the behaviour with string parsing, and it's only when the negative is on the RHS, as shown by the 4th example. 

If this isn't a bug/odd behaviour, what can I do to check if some expression parsed by latex is simplified?
This is problematic for me because `3x-1` is considered not simplified due to the rearrangement.

**Comment by gerryfletch** [comment]
Actually I'm relatively sure I can just use `count_ops(a) <= count_ops(simplified(b))`. Nevermind :)

**Comment by asmeurer** [comment]
This does look like a bug 

```
>>> print(parse_latex('3x-1'))
3*x - 1*1
```

**Comment by cocolato** [comment]
I opened a pr https://github.com/sympy/sympy/pull/24096
 to solve this problem, do you have any suggestions?
